specs/eip4844/p2p-interface.md

```diff
@@ -188,16 +215,22 @@ Request Content:
 ```
 
 Response Content:
+
+```python
+class BlobSidecars(Container):
```

Comment:
+1 on `AggregateBlobSidecar`

Reply:
`BlobsSidecar` would be multiple blobs in a `Sidecar` - I take `Sidecar` to mean the fluff around each blob (index, proof etc). Otherwise, see above for thoughts on `Aggregate`.

Reply:
resolving as dupe